src/Tribe/Service_Providers/Context.php

```diff
@@ -44,32 +45,54 @@ public function register() {
 	 * @return array The modified context locations.
 	 */
 	public function filter_context_locations( array $locations = [] ) {
+		$view_query_var = View::query_var();
+
 		$locations = array_merge(
 			$locations,
 			[
 				'event_display'        => [
 					'read'  => [
 						Tribe__Context::WP_MATCHED_QUERY => [ 'eventDisplay' ],
 						Tribe__Context::WP_PARSED        => [ 'eventDisplay' ],
-						Tribe__Context::REQUEST_VAR      => [ 'view', 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
+						Tribe__Context::REQUEST_VAR      => [
+							$view_query_var,
+							'tribe_view',
+							'tribe_event_display',
+							'eventDisplay'
+						],
 						Tribe__Context::QUERY_VAR        => 'eventDisplay',
 						Tribe__Context::TRIBE_OPTION     => 'viewOption',
 					],
 					'write' => [
-						Tribe__Context::REQUEST_VAR => [ 'view', 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
+						Tribe__Context::REQUEST_VAR => [
+							$view_query_var,
+							'tribe_view',
+							'tribe_event_display',
+							'eventDisplay'
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: c5487174b8603dee7dd11a6dd002ab31</sub>